cmd/printer.go

```diff
@@ -677,6 +684,38 @@ func (m MetalMachineTablePrinter) Print(data []*models.V1MachineResponse) {
 	m.render()
 }
 
+// Print a MetalSize in a table
+func (m MetalMachineIssuesTablePrinter) Print(data MachineIssues) {
+	for machine, is := range data {
+		id := ""
+		if machine.ID != nil {
+			id = *machine.ID
+		}
+		name := ""
+		if machine.Allocation != nil && machine.Allocation.Name != nil {
+			name = truncate(*machine.Allocation.Name, "...", 30)
+		}
+		partition := ""
+		if machine.Partition != nil && machine.Partition.ID != nil {
+			partition = *machine.Partition.ID
+		}
+		project := ""
+		if machine.Allocation != nil && machine.Allocation.Project != nil {
+			project = *machine.Allocation.Project
+		}
+
+		issues := ""
+		for _, issue := range is {
```

Comment:
better with strings.Join ?

Reply:
But how to prepend each issue with a hyphen? Somehow this code does not even work. Either with join nor with the loop I see any line breaks. :(